maven-core/src/main/java/org/apache/maven/plugin/internal/MojoLogWrapper.java

```diff
@@ -34,7 +34,9 @@ public MojoLogWrapper(Logger logger) {
     }
 
     public void debug(CharSequence content) {
-        logger.debug(toString(content));
+        if (isDebugEnabled()) {
```

Comment:
I notice this duplicates a lot of content from DefaultLog above. I wonder if that's fixable somehow? Not necessarily in this PR.